### ISSUE TITLE
Button missing a colon in CSS

### ISSUE THREAD

**Issue by SuNaden** [mxstbr/sharing#50]
The active and hover styles do not get applied to the email button, because there is simply a missing colon after the `:hover`

```
.resp-sharing-button--email:hover
.resp-sharing-button--email:active {
  background-color: #2B2B2B;
  border-color: #2B2B2B;
}
```

After checking the generated CSS, this is also an issue in the Xing button.

**Comment by mxstbr** [comment]
Would appreciate a PR fixing that!

**Comment by mxstbr** [comment]
(though I suspect this might've been fixed with #48)

**Comment by SuNaden** [comment]
Oh sorry, did not notice that PR. Yes #48 will surely fix it.
